lib/ops/opsservice/install.go

```diff
@@ -886,6 +886,10 @@ func (s *site) newProvisioningToken(operation ops.SiteOperation) (token string,
 		OperationID: operation.ID,
 		UserEmail:   agentUser.GetName(),
 	}
+	if operation.Type == ops.OperationExpand {
+		// Set a TTL for expand provisioning token.
```

Comment:
Nit, comment on why, not what. So in the comment, link to the issue or problem or regression, and say the TTL is set because this is required by teleport, etc, otherwise it would be very easy to come along and make the same mistake again.